content/00.front-matter.md

```diff
@@ -2,14 +2,22 @@
 
 {% for author in authors %}
 + **{{author.full_name}}**<br>
+  ![ORCID icon](images/orcid.svg){height="13px"}
```

Comment:
Adding the icon makes the soft requirement that all authors have an ORCID stronger.  The icon looks strange without the id.

I'm okay with that, but let's add `0000-0003-3877-0433` for @evancofer to `metadata.yaml`.